### ISSUE TITLE
yarn install failed install vscode-sqlite3

### ISSUE THREAD

**Issue by GongT** [Microsoft/vscode#61468]
This may relate to #61274 , but I cannot comment it with an unknown error, so post a new issue here.

I used `yarn global add windows-build-tools`, and everything works well.    
**Today**, after I run `git clean -x`, `yarn install` cannot success again.    
It says `vscode-sqlite3` won't build, because `python` not able to run.

My python is installed by `windows-build-tools`, so it's not in PATH.     
After add `c:\user\me\.windows-build-tools\python27`, install success.

**Comment by joaomoreno** [comment]
What's the error you're getting? Can you show me the full log?

**Comment by bpasero** [comment]
@joaomoreno yeah I thought you had the same issue

**Comment by GongT** [comment]
I do not have a log now, my build has already success, not saving the log.

**Comment by GongT** [comment]
To ensure everyone can run, I have created a script to set EVERY env var Explicitly (and remove All unused).    
There is no issue at least several month..

**Comment by joaomoreno** [comment]
Good to know you're unblocked now!
